From: Fabricio Silva <hi@fabricio.dev>
Date: Mon, 7 Aug 2023 09:29:47 +0100
Subject: [PATCH] Disable GitVersion

GitVersion is a dependency that needs git history to return the current version.
APKBUILD does not use git history. Disable it and just replace version in prepare() step.

---
diff --git a/src/Recyclarr.Cli/Console/Setup/ProgramInformationDisplayTask.cs b/src/Recyclarr.Cli/Console/Setup/ProgramInformationDisplayTask.cs
index 718c9afa..9780723b 100644
--- a/src/Recyclarr.Cli/Console/Setup/ProgramInformationDisplayTask.cs
+++ b/src/Recyclarr.Cli/Console/Setup/ProgramInformationDisplayTask.cs
@@ -6,7 +6,7 @@ public class ProgramInformationDisplayTask(ILogger log, IAppPaths paths) : IGlob
 {
     public void OnStart()
     {
-        log.Debug("Recyclarr Version: {Version}", GitVersionInformation.InformationalVersion);
+        log.Debug("Recyclarr Version: {Version}");
         log.Debug("App Data Dir: {AppData}", paths.AppDataDirectory);
     }
 }
diff --git a/src/Recyclarr.Cli/Program.cs b/src/Recyclarr.Cli/Program.cs
index da4a72c1..d9e133ef 100644
--- a/src/Recyclarr.Cli/Program.cs
+++ b/src/Recyclarr.Cli/Program.cs
@@ -25,7 +25,7 @@ internal static class Program

             config.SetApplicationName("recyclarr");
             config.SetApplicationVersion(
-                $"v{GitVersionInformation.SemVer} ({GitVersionInformation.FullBuildMetaData})");
+                "v{Version}");

             config.SetExceptionHandler((ex, resolver) =>
             {
